### PR TITLE
Install java 11 on the AMIs.

### PR DESCRIPTION
**Pull request by varshavaradarajan**
@bdpiparva - I haven't verified by building the AMI. Let me know if this is alright. I'll do the same for the demo AMI.

### CODE REVIEW

**Comment by bdpiprava** [comment]
Changes looks good to me. However, for centos 7 openjdk 11 is now available through yum.

File: go-server-packer.json [hunk 1]
```diff
@@ -43,9 +44,13 @@
         "sudo yum update -y",
         "sudo yum install -y java-1.8.0-openjdk-headless git mercurial subversion openssh-clients bash wget",
```
Comment (bdpiprava) [inline go-server-packer.json]: Do remove jdk 1.8.

**Review by bdpiprava — CHANGES_REQUESTED** [review]

**Comment by varshavaradarajan** [comment]
> for centos 7 openjdk 11 is now avay through yum.

Not available on amazon linux, @bdpiparva . Did `yum list --showduplicates java*`. Showed only java 7 & 8.

File: go-server-demo-packer.json [hunk 2]
```diff
@@ -3,7 +3,9 @@
     "gocd_version": "",
     "instance_type": "c5.large",
     "region": "{{env `REGION`}}",
-    "extra_ami_region_to_copy_to": "{{env `EXTRA_AMI_REGION_TO_COPY_TO`}}"
+    "extra_ami_region_to_copy_to": "{{env `EXTRA_AMI_REGION_TO_COPY_TO`}}",
+    "openjdk_url": "https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_linux-x64_bin.tar.gz",
```
Comment (bdpiprava) [inline go-server-demo-packer.json]: Need to update openjdk url to `https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz`

**Review by bdpiprava — APPROVED** [review]

**Pull request merged**